src/Renderer/Image.php

```diff
@@ -498,7 +498,10 @@ protected function drawText($text, $size, $position, $font, $color, $alignment =
     protected function imageFilledPolygonWrapper($image, array $points, $numPoints, $color, $filled)
     {
         if (! $filled) {
-            return imagepolygon($this->resource, $points, $numPoints, $color);
+            if (PHP_VERSION_ID < 80100) {
+                return imagepolygon($this->resource, $points, $numPoints, $color);
+            }
```

Comment:
This part is not necessary as PHP version 8.1 is the minimum for this package:

https://github.com/laminas/laminas-barcode/blob/e8e85eb77c0b9768846e7e97a285485790e4f942/composer.json#L27-L28